tests/bluetooth/tester/src/audio/btp_pbp.c

```diff
@@ -0,0 +1,267 @@
+/* btp_pbp.c - Bluetooth PBP Tester */
+
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "btp/btp.h"
+
+#include <zephyr/bluetooth/audio/pbp.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util.h>
+#define LOG_MODULE_NAME bttester_pbp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
+
+#define PBP_EXT_ADV_METADATA_LEN_MAX 128
+
+static uint8_t pbp_features_cached;
+static uint8_t pbp_metadata_cached_len;
+static uint8_t pbp_metadata_cached[PBP_EXT_ADV_METADATA_LEN_MAX];
+static uint8_t pbp_name_cached_len;
+static uint8_t pbp_broadcast_name_cached[BT_AUDIO_BROADCAST_NAME_LEN_MAX];
+
+static bool scan_get_broadcast_name_len(struct bt_data *data, void *user_data)
+{
+	uint8_t *broadcast_name_len = user_data;
+
+	switch (data->type) {
+	case BT_DATA_BROADCAST_NAME:
+		*broadcast_name_len = data->data_len;
+		return false;
```

Comment:
It returns false when the name is there. Sounds inverted to me.

Reply:
Returning `false` here means that it will stop the parsing. Since this callback only handles the name, this is correct, even if it's not obvious :D